samples/bluetooth/cap_initiator/src/cap_initiator_broadcast.c

```diff
@@ -133,9 +133,10 @@ static int setup_extended_adv_data(struct bt_cap_broadcast_source *source,
 	uint32_t broadcast_id;
 	int err;
 
-	err = bt_cap_initiator_broadcast_get_id(source, &broadcast_id);
+	err = bt_rand(&broadcast_id, BT_AUDIO_BROADCAST_ID_SIZE);
```

Comment:
Why did this sample not get the Kconfig options?

Reply:
It should've, Ill add it